README.md

```diff
@@ -257,6 +257,13 @@ With the newly arrived data, BayBE can produce a refined design for the next ite
 This loop would typically continue until a desired target value has been achieved in

```

Comment:
did you do anything wird? Seems this PR hasnt triggered the CI ??

Reply:
Not that I am aware of :O